src/ontology/mondo-ingest.Makefile

```diff
@@ -598,6 +610,19 @@ update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+
+#################
+##### Ad hoc ####
+#################
+tmp/ordo-subsets.tsv: $(COMPONENTSDIR)/ordo.owl
+	$(ROBOT) query -i $< --query ../sparql/select-ordo-subsets.sparql $@
+
+reports/ordo-subsets.robot.template.tsv: tmp/ordo-subsets.tsv tmp/mondo.sssom.tsv
+	python3 $(SCRIPTSDIR)/ordo_subsets.py \
+	--mondo-mappings-path tmp/mondo.sssom.tsv \
```

Comment:
Is this getting the latest `mondo.sssom.tsv` file?

Reply:
## This shouldn't have been part of the diff
@twhetzel Sorry, this is a wonky PR situation. That change is not part of this PR. It's part of #510, the branch for which this PR was used as a base.

I originally set this PR to be merged into `develop`, because that is where we do want to merge it. However, I forgot that this would make it so that all of the other changes in #510 would also show up here. I've now changed it so that this will merge into #510, so now the diff that shows is correct. The workflow we'll want to do, I think, is: (1) merge #510 into `develop` (we're just waiting on Nico's final explicit approval for that), and then I can (2) change the base for this to merge into `develop`.

Reply:
no worries. i just wanted to make sure that the new SOP/goal in https://github.com/monarch-initiative/mondo-ingest/issues/529 is accounted for here in `reports/ordo-subsets.robot.template.tsv` so that it also has the latest `mondo.sssom.tsv` file

Reply:
## Answer to the question
In any case, as per your original question, there are several make goals which depend on `tmp/mondo.sssom.tsv`. You bring up a good point, related to #529. The way this dependency is set up, simply by listing `tmp/mondo.sssom.tsv` in the prerequisites of the goal, is the typical way to do it in `make`. However, in the event that the `mondo` repo has changed since `tmp/mondo.sssom.tsv` was generated, `make` will have no way of knowing that, and will think that `tmp/mondo.sssom.tsv` is up-to-date, and use the one that's presently there on the system.

Perhaps we're fine with that being the behavior. During a fresh build, this will not be a problem. It'll only be a problem in development situations. If we want to fix this for those situations as well, then, as far as I see it, we have 2 options.

Both options involve at least this change:
Move dependency into the goal's body. In this example, that would change it to:
```make
reports/ordo-subsets.robot.template.tsv: tmp/ordo-subsets.tsv 
	$(MAKE) tmp/mondo.sssom.tsv
	python3 $(SCRIPTSDIR)/ordo_subsets.py \
	...
```

### a. Always run a full refresh of `tmp/mondo.sssom.tsv`
Always run a refresh each time a goal is run which has it as a dependency.

This involves nothing else in addition to the change I just described above.

This could be be a real damper on local development. Not always terribly so, though. When I'm developing a goal that uses Python, I usually do all of my iteration in the debugger, and then run the make goal once at the end just to make sure it works.

### b. Intelligently refresh `tmp/mondo.sssom.tsv` when needed
This would be cool to do at some point. Basically, in addition to the change I described above, we'd have an if statement at the top of the goal for `tmp/mondo.sssom.tsv` or `tmp/mondo_repo_built`. It would compare the latest git commit hashes on `master` for  the cloned `mondo` dir in `tmp/` and the actual repo on GitHub. If they're the same, do nothing. Else, it will re-run `tmp/mondo_repo_built` --> `tmp/mondo.sssom.tsv`.

---

I think this merits its own GH issue.

Reply:
I agree this could use it's own GH issue and maybe a ~todo/TODO for the line in this PR to evaluate the outcome of the future ticket

Reply:
@twhetzel Sounds good! I created:
- #538 

However, I didn't add a `todo` because this is more pervasive; there are 4 goals that use `tmp/mondo.sssom.tsv` as a dependency. If you want I can insert a `todo` on each of them.

Reply:
RE: `todo` - whichever works better for your tracking